Aspire BudgetingTests/GoogleDriveManagerTests.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  GoogleDriveManagerTests.swift
+//  Aspire BudgetingTests
+//
+//  Created by TeraMo Labs on 10/25/19.
+//  Copyright © 2019 TeraMo Labs. All rights reserved.
+//
+
+import XCTest
+import GTMSessionFetcher
+import GoogleAPIClientForREST
+import GoogleSignIn
+import Combine
+
+@testable import Aspire_Budgeting
+class MockAuthorizer: NSObject, GTMFetcherAuthorizationProtocol {
+  func authorizeRequest(_ request: NSMutableURLRequest?, delegate: Any, didFinish sel: Selector) {
+    
+  }
+  
+  func stopAuthorization() {
+    
+  }
+  
+  func stopAuthorization(for request: URLRequest) {
+    
+  }
+  
+  func isAuthorizingRequest(_ request: URLRequest) -> Bool {
+    return false
+  }
+  
+  func isAuthorizedRequest(_ request: URLRequest) -> Bool {
+    return false
+  }
+  
+  var userEmail: String?
+  
+  
+}
+
+class GoogleDriveManagerTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/c2420c127e412148682bc0fe81f219ec96ee8752/Aspire BudgetingTests/GoogleDriveManagerTests.swift#L42"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>